src/features/forecastapi/forecast-api.ts

```diff
@@ -13,17 +13,23 @@ namespace ForecastApiUtilities {
 
   /**
    * Gets non project time types from forecast
-   * 
+   *
    * @returns All non project times where is_internal_time is false
    */
   export const getNonProjectTime = async (): Promise<NonProjectTime[]> => {
     try {
       const result = await fetch(`${process.env.forecast_base_url}non_project_time`, { headers: headers });
-      const NonProjectTimes: NonProjectTime[] = await result.json();
-      return NonProjectTimes.filter(NonProjectTime => NonProjectTime.is_internal_time !== true);
+      const nonProjectTimes = await result.json();
+      if (nonProjectTimes[0].id){
+        const filterRes: NonProjectTime[] = nonProjectTimes.filter(nonPTime => nonPTime.is_internal_time !== true);
```

Comment:
In typescript you can use ! to check if something is false/undefined instead of specifically checking for false/true.
```suggestion
        const filterRes: NonProjectTime[] = nonProjectTimes.filter(nonPTime => !nonPTime.is_internal_time);
```

Reply:
understood